tests/scripts/all.sh

```diff
@@ -207,13 +196,6 @@ General options:
 Tool path options:
      --armc5-bin-dir=<ARMC5_bin_dir_path>       ARM Compiler 5 bin directory.
      --armc6-bin-dir=<ARMC6_bin_dir_path>       ARM Compiler 6 bin directory.
-     --gnutls-cli=<GnuTLS_cli_path>             GnuTLS client executable to use for most tests.
```

Comment:
Should we recognize, but ignore these options for the time being, for the sake of facilitating CI scripts that apply to both before and after or that apply to both Mbed Crypto and Mbed TLS?

Reply:
Yes, these need to be handled. They don't need to be documented.